src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write code here
+const allLiElement = [ ...document.querySelectorAll('li') ];
+const dateList = [];
+const accessToUlTag = document.querySelector('ul');
```

Comment:
```suggestion
const list = document.querySelector('ul');
```

or something like that